app/templates/settings/applications.hbs

```diff
@@ -0,0 +1,21 @@
+<div class="row">
```

Comment:
No exactly @sumedh123 . We want the tabs to be placed on the parent template. Which is in this case `settings.hbs`. (If you cannot find that, run `ember generate route settings`).

See `events.hbs`, `events/live.hbs`, `events/draft.hbs` and checkout how it's done there.